### PR TITLE
Image.list info update

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
**Description of the Pull Request (PR):**

In order to align the current info print from instance.list
Replaced:
```bash  
  $ singularity instance.list
    /home/mibauer/singularity/sinstance/test.img  [2056.132716]  Instances:
         Instance PID:   11963
         Instance Name:  test
         
    $ sudo singularity instance.list -u mibauer
    /home/mibauer/singularity/sinstance/test.img  [2056.132716]  Instances:
         Instance PID:   16219
         Instance Name:  test
         
         Instance PID:   11963
         Instance Name:  test
```

For:

```bash
    $ singularity instance.list
    DAEMON NAME      PID      CONTAINER IMAGE
    test            11963     /home/mibauer/singularity/sinstance/test.img
         
    $ sudo singularity instance.list -u mibauer
        DAEMON NAME      PID      CONTAINER IMAGE
    test            11963     /home/mibauer/singularity/sinstance/test.img
    test            16219     /home/mibauer/singularity/sinstance/test.img
```

**Checkoff for all PRs:**

- [x] I have read the [Guidelines for Contributing](https://github.com/singularityware/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- [ ] I have added changes to the [CHANGELOG](https://github.com/singularityware/singularity/blob/development/CHANGELOG.md) and and documentation updates to the [singularityware](https://www.github.com/singularityware/singularityware.github.io) documentation base.
- [x] I have tested this PR locally with a `make test`
- [x] This PR is NOT against the project's master branch
- [x] I have added myself as a contributor to the [contributors's file](https://github.com/singularityware/singularity/blob/master/CONTRIBUTORS.md)
- [x] This PR is ready for review and/or merge

Attn: @singularityware-admin

### CODE REVIEW

File: libexec/cli/instance.list.info [hunk 1]
```diff
@@ -14,19 +14,13 @@ LIST OPTIONS:
...
     $ sudo singularity instance.list -u mibauer
-    /home/mibauer/singularity/sinstance/test.img  [2056.132716]  Instances:
-         Instance PID:   16219
-         Instance Name:  test
-         
-         Instance PID:   11963
-         Instance Name:  test
-
-
+        DAEMON NAME      PID      CONTAINER IMAGE
```
Comment (GodloveD) [inline libexec/cli/instance.list.info]: I think a stray tab must have snuck into this line.

File: libexec/cli/instance.list.info [hunk 2]
```diff
@@ -14,19 +14,13 @@ LIST OPTIONS:
...
+    test            11963     /home/mibauer/singularity/sinstance/test.img
+    test            16219     /home/mibauer/singularity/sinstance/test.img
```
Comment (GodloveD) [inline libexec/cli/instance.list.info]: This still doesn't seem completely realistic.  I don't think two instances can share the same name.  I think the second line should read something like:
```
test2           16219     /home/mibauer/singularity/sinstance/test.img
```

**Review by GodloveD — CHANGES_REQUESTED** [review]
Thanks very much for looking through the help docs and working to keep us honest.  It's much appreciated.  I think there are still a few changes that need to be made to work this in.

**Comment by ArangoGutierrez** [comment]
Changes proposed by @GodloveD on commit #b628c1d

**Review by GodloveD — APPROVED** [review]
Looks good to me!

**Comment by GodloveD** [comment]
@bauerm97 can you please review this very small change to the documentation and let me know if it looks good to you?  If it does I will merge.  Thank you.

**Comment by bauerm97** [comment]
@GodloveD looks good to me!

**Comment by GodloveD** [comment]
Thanks @ArangoGutierrez !

**Pull request merged**